tools/depends/target/openssl/Makefile

```diff
@@ -4,15 +4,27 @@ DEPS = ../../Makefile.include Makefile OPENSSL-VERSION ../../download-files.incl
 
 # configuration settings
 ifeq ($(OS), linux)
-  # Need to export our vars to override "default" conf data of openssl
   TARGETOPT=--with-zlib-include=$(PREFIX)/include --with-zlib-lib=$(PREFIX)/lib
-  CONFIGURE=MACHINE=$(PLATFORM) ./config no-shared zlib no-tests no-asm $(TARGETOPT) --prefix=$(PREFIX) --libdir=lib
+  CONFIGURE=./Configure no-shared zlib no-tests no-asm no-module  $(TARGETOPT) --prefix=$(PREFIX) --libdir=lib
+  ifeq ($(TARGET_PLATFORM),webos)
+    CONFIGURE+= linux-armv4
```

Comment:
Do we need the special handling for webOS here? Currently `CPU=armeabi-v7a` on webOS. I looked for usages of `armeabi-v7a` in `tools/depends` and it seems that it's not really used anywhere. I think we can safely set https://github.com/xbmc/xbmc/blob/29e78e56830187b8950afc401404b4f86add7ac4/tools/depends/configure.ac#L363 to `arm` and remove this specific check here.

Reply:
I might leave that for you in a follow up. I cant personally test/build, and just dont want to get stuck in a rabbit hole i cant fix.
Your reasoning sounds sane, so fingers crossed its that easy.